packages/dev/core/src/Behaviors/Meshes/fadeInOutBehavior.ts

```diff
@@ -2,6 +2,8 @@ import type { Behavior } from "../behavior";
 import type { AbstractMesh } from "../../Meshes/abstractMesh";
 import type { Mesh } from "../../Meshes/mesh";
 import type { Nullable } from "../../types";
+import { Observer } from "core/Misc";
+import { Scene } from "core/scene"

```

Comment:
should be `import type`  instead of import (you can get that auto-fixed in vs-code, otherwise - just be sure you run the formatting and linter checks)